gradle.properties

```diff
@@ -15,4 +15,4 @@ systemProp.org.gradle.internal.publish.checksums.insecure=true
 # removed when/if this is resolved on the gradle side.
 #
 # Gradle also needs more heap space to build this project than the default 1GB.
-org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m -Xmx2048m
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m -Xmx2048m -Xss512m
```

Comment:
Please add a comment to the comment block above that explains the need for the `Xss` option (similar to the existing comments about the `XX:MaxMetaspaceSize` and `Xmx` options.

Also, 512m seems quite large. The default on most platforms is 1m. I'm not too familiar with JVM memory tuning -- do you think it's worth starting with a smaller number?

Reply:
Decresed it to 32m, still works reliably on my machine. Also added a comment.